client/src/components/share/share-template.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { faTwitter } from '@fortawesome/free-brands-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { ShareRedirectProps } from './types';
+
+export const ShareTemplate: React.ComponentType<ShareRedirectProps> = ({
+  redirectURL
+}) => {
+  const { t } = useTranslation();
+  return (
+    <a
+      title={t('buttons.share-on-twitter') || 'Share on Twitter'}
+      data-testid='ShareTemplateWrapperTestID'
+      className='btn fade-in'
+      href={redirectURL}
+      target='_blank'
+      rel='noreferrer'
+    >
+      <span className='sr-only'>
+        {t('buttons.share-on-twitter')}, {t('aria.opens-new-window')}
+      </span>
+      <FontAwesomeIcon
+        icon={faTwitter}
+        size='1x'
+        aria-label='twitterIcon'
+        aria-hidden='true'
+      />
+      <span aria-hidden>{t('buttons.tweet')}</span>
```

Comment:
```suggestion
      {t('buttons.tweet')} <span className='sr-only'>
        {t('aria.opens-new-window')}
      </span>
```

If the visual text of this button is just going to be "Tweet" then the accessible name should be the same (except we also need to add "Opens in new window").

Reply:
updated

Reply:
tested it working good after adding this change